### PR TITLE
TBG: .tpl no `_profile` suffix

### PR DESCRIPTION
**Pull request by ax3l**
change the `_profile.tpl` suffix to `.tpl` in files with legacy naming.

Reduces confusion with `picongpu.profile` "profiles" and had no real meaning in the TBG templates.

Documentation and references to files have been updated accordingly.

### CODE REVIEW

**Comment by ax3l** [comment]
ready for merge! :)

**Comment by ax3l** [comment]
ping @PrometheusPi

File: etc/picongpu/lawrencium-lbnl/picongpu.profile.example [hunk 1]
```diff
@@ -56,6 +56,6 @@ export MY_NAME="$(whoami) <$MY_MAIL>"
 
 # "tbg" default options #######################################################
 #   - SLURM (sbatch)
-#   - fermi queue (also available: 2 K20 via k20_profile.tpl)
+#   - fermi queue (also available: 2 K20 via k20.tpl)
 export TBG_SUBMIT="sbatch"
-export TBG_TPLFILE="etc/picongpu/lawrencium-lbnl/fermi_profile.tpl"
+export TBG_TPLFILE="etc/picongpu/lawrencium-lbnl/fermi.tpl"
```
Comment (PrometheusPi) [inline etc/picongpu/lawrencium-lbnl/picongpu.profile.example]: QUESTION:
From the diff I just see deletion of `k20_profile.tpl` and `fermi_profile.tpl` no renaming - were these accidentally deleted or were there new profiles before?

Comment (ax3l) [reply etc/picongpu/lawrencium-lbnl/picongpu.profile.example]: huh, weird. good catch, will double-check and add again if necessary!

Comment (ax3l) [reply etc/picongpu/lawrencium-lbnl/picongpu.profile.example]: fixed! thank you!

Comment (PrometheusPi) [reply etc/picongpu/lawrencium-lbnl/picongpu.profile.example]: :+1:

**Pull request merged**